contributors/devel/sig-release/cherry-picks.md

```diff
@@ -32,22 +32,23 @@ branches.
 
 ## What Kind of PRs are Good for Cherry Picks
 
-Compared to the normal master branch's merge volume across time,
-the release branches see one or two orders of magnitude less PRs.
-This is because there is an order or two of magnitude higher scrutiny.
-Again, the emphasis is on critical bug fixes, e.g.,
+Patch releases must be easy and safe to consume, so security fixes
+and critical bugfixes can be delivered with minimal risk of regression.
 
-- Loss of data
-- Memory corruption
-- Panic, crash, hang
-- Security
+Only the following types of changes are expected to be backported:
 
-A bugfix for a functional issue (not a data loss or security issue) that only
-affects an alpha feature does not qualify as a critical bug fix.
+- Security fixes
+  - Dependency updates that just aim to silence some scanners
+    and do not fix any vulnerable code are **not** eligible for cherry-picks.
+- Regression fixes
+- Critical bug fixes (loss of data, memory corruption, panic, crash, hang)
+  - A bugfix for a functional issue (not a data loss or security issue) that only
+    affects an alpha feature does not qualify as a critical bug fix.
```

Comment:
This bullet point is a little contradictory to the previous bullet point ("Critical bug fixes"). This could imply that we accept cherry-picks fixing functional issues for beta and stable features, but is that really what we want? If it's not a regression, and it isn't loss of data, memory corruption, panic, crash, hang... I think we might want to apply the same treatment regardless of the feature stage.

Reply:
alpha features should to receive backports at any cost, the level of stability in alpha is very bad, some of the features have minimal testing and allowing backports will encourage wrong behaviors on users and developers, instead of working towards graduating they will ask for backports and fixing forward ... I feel very strong about this opinion, don't know if others think the same

I would go further and also include beta in the same boat, but the current situation it is what it is and we have betas feature that have more than 10 releases, don't know if we want now to evaluate the level of beta?

Reply:
If enabling an off-by-default alpha feature regresses something, we wouldn't consider that a regression we would backport for. If an on-by-default beta feature regresses something, we would.

Reply:
I like that

Reply:
pushed a commit clarifying that